modal/runner/containers/vllm_unified.py

```diff
@@ -139,19 +142,33 @@ def __init__(self):
     gpu=modal.gpu.A100(count=1, memory=40),
     concurrent_inputs=32,
 )
+
+# A re-mapping of model names to their respective quantized models.
+# From the outside, the model name is the original, but internally,
+# we use the quantized model name.
+QUANTIZED_MODELS = {
+    "NeverSleep/Noromaid-v0.1-mixtral-8x7b-Instruct-v3": "TheBloke/Noromaid-v0.1-mixtral-8x7b-Instruct-v3-GPTQ",
+    "jondurbin/bagel-34b-v0.2": "TheBloke/bagel-34b-v0.2-GPTQ",
+}
+
 VllmContainer_NeverSleepNoromaidMixtral8x7B = _make_container(
     name="VllmContainer_NeverSleepNoromaidMixtral8x7B",
-    model_name="NeverSleep/Noromaid-v0.1-mixtral-8x7b-Instruct-v3",
-    gpu=modal.gpu.A100(count=2, memory=80),
+    model_name="TheBloke/Noromaid-v0.1-mixtral-8x7b-Instruct-v3-GPTQ",
+    gpu=modal.gpu.A100(count=1, memory=40),
     concurrent_inputs=4,
     max_containers=3,
```

Comment:
open question as to whether we want to bump max containers  or adjust concurrent_inputs at all

a little hesitant to do much else rn, since it introduces more confounding variables to the change